### PR TITLE
Vitalii Bondarenko: html-form

### PR DESCRIPTION
**Pull request by VitaliyBondarenko1982**
[DEMO LINK](https://VitaliyBondarenko1982.github.io/layout_html-form/)
[TEST REPORT LINK](https://VitaliyBondarenko1982.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+.block-input {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.input-age {
+  padding-right: 7px;
+}
+
+.input-sleep {
```
Comment (irenhh) [inline src/style.css]:
```suggestion
.input-sleep-time {
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.block-form {
+  margin-bottom: 20px;
+  padding-bottom: 0;
+}
+
+.block-input {
```
Comment (irenhh) [inline src/style.css]:
```suggestion
.block-of-inputs {
```

Comment (VitaliyBondarenko1982) [reply src/style.css]: Виправив

File: src/style.css (hunk 1 above)
Comment (VitaliyBondarenko1982) [reply src/style.css]: виправив

**Pull request closed without merging**